### PR TITLE
add op type in check nan/inf

### PR DESCRIPTION
**Pull request by seiriosPlus**

### CODE REVIEW

File: paddle/fluid/framework/operator.cc [hunk 1]
```diff
@@ -882,7 +882,8 @@ class RuntimeInferShapeContext : public InferShapeContext {
   const RuntimeContext& ctx_;
 };
 
-static void CheckTensorNANOrInf(const std::string& name,
+static void CheckTensorNANOrInf(const std::string& type,
```
Comment (typhoonzero) [inline paddle/fluid/framework/operator.cc]: name it `op_type` should be easy to read

**Review by typhoonzero — DISMISSED** [review]

File: paddle/fluid/framework/operator.cc (hunk 1 above)
Comment (seiriosPlus) [reply paddle/fluid/framework/operator.cc]: done.

File: paddle/fluid/framework/operator.cc [hunk 2]
```diff
@@ -988,9 +989,10 @@ void OperatorWithKernel::RunImpl(const Scope& scope,
       auto* var = exec_scope.FindVar(vname);
       if (var == nullptr) continue;
       if (var->IsType<framework::LoDTensor>()) {
-        CheckTensorNANOrInf(vname, var->Get<framework::LoDTensor>());
+        CheckTensorNANOrInf(type_, vname, var->Get<framework::LoDTensor>());
       } else if (var->IsType<framework::SelectedRows>()) {
-        CheckTensorNANOrInf(vname, var->Get<framework::SelectedRows>().value());
+        CheckTensorNANOrInf(type_, vname,
+                            var->Get<framework::SelectedRows>().value());
```
Comment (chengduoZH) [inline paddle/fluid/framework/operator.cc:995]: In fact, if we set the `FLAGS_check_nan_inf=1` and `GLOG_vmodule=operator=4`, we can know the `nan/inf` is happend in which op.

Comment (seiriosPlus) [reply paddle/fluid/framework/operator.cc:995]: yes, I'm agreed. But it may print much more logs I will not expect.

**Review by chengduoZH — APPROVED** [review]

**Pull request merged**